lagunita/lms/templates/su-footer.html

```diff
@@ -23,10 +23,12 @@
           <li class="emergency alt"><a href="https://emergency.stanford.edu">Emergency Info</a></li>
         </ul>
         <ul id="policy-links">
-          <li class="terms alt"><a href="https://www.stanford.edu/site/terms/">Terms of Use</a></li>
```

Comment:
Where else is this `.terms` class being referenced?
Is it it in CSS?

Reply:
It isn't, that's why I'm removing it. The `alt` class is being used in the CSS to break each link in mobile though.